menu/menu_setting.c

```diff
@@ -9635,6 +9758,71 @@ static bool setting_append_list_input_player_options(
                parent_group);
          (*list)[list_info->index - 1].bind_type = i + MENU_SETTINGS_BIND_BEGIN;
       }
+      for (j = 0; j < RETROPAD_RETRO_SENSOR_LAST; j++){
+
+         char label[NAME_MAX_LENGTH];
+         char name[NAME_MAX_LENGTH];
+         strlcpy(label, msg_hash_to_str(MENU_ENUM_LABEL_VALUE_INPUT_SENSOR_FIRST+j),NAME_MAX_LENGTH);
+         snprintf(name,NAME_MAX_LENGTH,
+            "tmp name sensor bind %d", j);
```

Comment:
Was this part supposed to be temporary?

Reply:
Honestly I am not sure what the "name" parameter even does.